plugins/backup/dummy/src/main/java/org/apache/cloudstack/backup/DummyBackupProvider.java

```diff
@@ -86,13 +86,19 @@ public boolean restoreVMFromBackup(VirtualMachine vm, Backup backup) {
     @Override
     public Map<VirtualMachine, Backup.Metric> getBackupMetrics(Long zoneId, List<VirtualMachine> vms) {
         final Map<VirtualMachine, Backup.Metric> metrics = new HashMap<>();
-        final Backup.Metric metric = new Backup.Metric(1000L, 100L);
         if (vms == null || vms.isEmpty()) {
             return metrics;
         }
         for (VirtualMachine vm : vms) {
+            Backup.Metric metric = new Backup.Metric(1000L, 100L);
```

Comment:
Could we extract these values to variables? In my point of view, they seem like random numbers.